server/src/constants.ts

```diff
@@ -11,7 +11,7 @@ function getEnv(key: string): string {
   return process.env[key]!;
 }
 
-export const EXPRESS_PORT = process.env.EXPRESS_PORT || 3201;
+export const EXPRESS_PORT = 3201;
```

Comment:
```suggestion
export const EXPRESS_PORT = process.env.EXPRESS_PORT || 3201;
```

I think this is important to leave, especially for a possible production environment. People can deal with editing their `.env` file (which they have to do after #21 anyways)